### ISSUE TITLE
[4.0] Cannot read property 'addEventListener' of null - passwordview.min.js

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#18723]
### Steps to reproduce the issue

Install 4.0 alpha 2
in Admin Users Options -> set Display Labels = Text (thats the green and not grey option of the incredibly difficult to understand toggle boxes!) 
Go to the frontend and reload the home page (which has a login module by default on the right)

### Expected result

No JS errors in console.

### Actual result

Error message in console 

```
Uncaught TypeError: Cannot read property 'addEventListener' of null
    at HTMLDocument.<anonymous> (passwordview.min.js?027cdb66b159c72b9497be415bbbd127:1)
```

![screen shot 2017-11-19 at 19 21 42](https://user-images.githubusercontent.com/400092/32994408-e496534e-cd5e-11e7-80c5-5a13dd0fb7a0.png)

**Comment by ghost** [comment]
Issue confirmed. Set on Module "Login Form" the "Display Labels".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18723">issues.joomla.org/tracker/joomla-cms/18723</a>.</sub>

**Comment by dgrammatiko** [comment]
Javascript team is working on another approach for ALL fields that are js dependant so that they'll be implemented as custom elements (check the user field and the media field). So this is WIP  and again can be marked as known issue

**Comment by ghost** [comment]
We need Information what to test, what not now. I suggest on Download-Page like https://github.com/joomla/joomla-cms/releases/tag/4.0.0-alpha2

**Comment by PhilETaylor** [comment]
Yes, simply saying this is a "known issue" because you have some secret plans that you are not sharing with anyone is simply a waste of testers time.... 

There is nothing about these secret plans on 
- https://www.joomla.org/announcements/release-news/5718-joomla-4-0-alpha-1-release.html
- https://docs.joomla.org/Category:Version_4.0_Alpha_1_FAQ
- https://docs.joomla.org/Potential_backward_compatibility_issues_in_Joomla_4

**Comment by dgrammatiko** [comment]
@PhilETaylor there is nothing secret, so please stop repeating this. I've mentioned the 2 fields that are already implemented in j4 and since you're asking here is the **public** code that we're working on:
https://codepen.io/dgt41/

**Comment by PhilETaylor** [comment]
I was asked to test the Alpha, I did. This (and multiple other JS issues) is a problem with that has been confirmed with the alpha.

There was nothing about this being a "known issue" - its only "known" to you, it is not on a list of known issues... This "known issues" list should be a public document, or at least on one of the urls above which is the correct place for known issues to be listed. 

You cannot just close issues stating YOU alone know about things... 

These issues should remain open until the "javascript team" implement their secrets, and confirm that the feature/bug/report reported is either a) No longer relevant, or b) fixed. 

Closing issues and simply ignoring them is what happens else.

**Comment by brianteeman** [comment]
actually the request was to test alpha with extensions 
> **There is one goal for this release:** to allow advance testing of custom extensions so that extension developers can report any problems well before the final release.

**Comment by dgrammatiko** [comment]
@PhilETaylor where did I state that this issue **should** be closed?
I just said that this should be marked as known issue and tried to justified the reason for that...

**Comment by PhilETaylor** [comment]
Its funny how every conversation with @dgt41 turns into this pathetic tirade. I tell you what - I'll NEVER test JS or Custom Elements ever again. Job solved.

**Comment by PhilETaylor** [comment]
> actually the request was to test alpha with extensions

tldr; I guess, as I have not even installed an extension to an alpha...

**Comment by PhilETaylor** [comment]
> here is the public code that we're working on

Is there a Public, Joomla user hosted, GitHub Repository for this?

**Comment by dgrammatiko** [comment]
Yes there is: https://github.com/joomla/joomla-cms/tree/4.0-dev/build/webcomponents

**Comment by C-Lodder** [comment]
PR: https://github.com/joomla/joomla-cms/pull/18755

**Comment by ghost** [comment]
closed as having Pull Request #18755
